packages/react-hooks/README.md

```diff
@@ -84,3 +84,14 @@ function MockComponent() {
   );
 }
 ```
+
+### `usePrevious()`
+
+This hook will store the previous value of a given variable.
+
+```tsx
+function MockComponent({value}) {
+  const previousValue = usePrevious(value);
+  return null;
```

Comment:
I'd love to see a more realistic example here. Something that shows a very minimal real use-case.